user_profile.py

```diff
@@ -298,7 +347,9 @@ def post(self):
                         if result:
                             abort(
                                 403,
-                                message=f"This user with {user_id} already existed in the database!",
+                                {
+                                    'message' : f"This user with {user_id} already existed in the database!"
+                                }
```

Comment:
Hmm for security reason, let's not expose the `user_id` here. We only need to tell the client user existed.

```suggestion
                                    'message' : f"User already exist"
                                }
```

No need to mention database.